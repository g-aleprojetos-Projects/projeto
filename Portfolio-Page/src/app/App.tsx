import manutencao from './../assets/images/manutencao.svg';
import * as S from './App.styles';

export function App() {
  return (
    <S.Container data-testid={'pagina-app'}>
      <S.Image
        data-testid={'image-manutencao'}
        src={manutencao}
        alt="Página em Manutenção"
      />
    </S.Container>
  );
}
