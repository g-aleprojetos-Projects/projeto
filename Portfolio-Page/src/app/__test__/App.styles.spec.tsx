import renderer from 'react-test-renderer';
import 'jest-styled-components';
import * as S from '../App.styles';

describe('App.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const container = renderer.create(<S.Container />).toJSON();
    expect(container).toMatchSnapshot();
  });
  test('Image DEVE ser igual ao snapshot', () => {
    const image = renderer.create(<S.Image />).toJSON();
    expect(image).toMatchSnapshot();
  });
});
