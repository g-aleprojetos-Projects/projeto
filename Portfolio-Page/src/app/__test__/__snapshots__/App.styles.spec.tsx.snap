// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.styles Container DEVE ser igual ao snapshot 1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 100dvw;
  height: 100dvh;
  background-color: #121214;
}

<div
  className="c0"
/>
`;

exports[`App.styles Image DEVE ser igual ao snapshot 1`] = `
.c0 {
  width: 40em;
  height: 40em;
}

<img
  className="c0"
/>
`;
